pipeline-scripts/buildlib.groovy

```diff
@@ -624,16 +624,16 @@ def args_to_string(Object... args) {
 }
 
 /**
- * We need to execute some scripts directly from the rcm-guest host. To perform
+ * We need to execute some scripts directly from the ocp-artifacts host. To perform
  * those operations, we stream the script into stdin of an SSH bash invocation.
  * @param git_script_filename  The file in build-scripts/rcm-guest to execute
  * @param args A list of arguments to pass to script
  * @return Returns the stdout of the operation
  */
-def invoke_on_rcm_guest(git_script_filename, Object... args ) {
+def invoke_on_ocp_artifacts(git_script_filename, Object... args ) {
```

Comment:
In the era of rcm-guest, we run `push.enterprise.sh` on rcm-guest to publish a local directory to the old mirror. Now both the old mirror and rcm-guest is decommissioned. Although we host plashet repos on ocp-artifacts, I don't see a reason to invoke any commands on ocp-artifacts.

Reply:
We still run `public-oc-binary.sh` though. This script seems to store `oc` for different arches on the server it runs on. We used to run it on rcm-guest, where should we run it instead?

Reply:
We needed a jump host in the past for one reason or the other. That reason does not exist anymore. I think we can run everything on buildvm, and then copy the resulting artifacts over to ocp-artifacts / mirror with the existing means.

Reply:
I think we can remove that script. OC binaries are already being pushed to our [s3 bucket](https://s3.console.aws.amazon.com/s3/buckets/art-srv-enterprise?region=us-east-1&prefix=pub/openshift-v3/clients/3.11.213/linux/&showversions=false) by [this script](https://github.com/openshift-eng/aos-cd-jobs/blob/master/jobs/build/ocp3/s3-publish-oc-binary.sh). Since we'd have to tell `oc` consumers to change the location they're pulling from, we can instruct them to use our AWS S3 bucket. We might need to create a user for that, but this is something I'd address when the need comes.

Done it in c93e14f66ec23d9a8b15977ef4c5ec5be60ca57f